libs/application/templates/parental-leave/src/fields/InReviewSteps/EditsInReviewSteps.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC } from 'react'
+
+import { useLocale } from '@island.is/localization'
+import format from 'date-fns/format'
+
+import { FieldBaseProps } from '@island.is/application/core'
+import { Box, Text } from '@island.is/island-ui/core'
+import ReviewSection, { ReviewSectionState } from './ReviewSection'
+
+import { parentalLeaveFormMessages } from '../../lib/messages'
+
+import { getExpectedDateOfBirth } from '../../parentalLeaveUtils'
+
+import { States as ApplicationStates } from '../../lib/ParentalLeaveTemplate'
+type StateMapEntry = { [key: string]: ReviewSectionState }
+type StatesMap = {
+  employer: StateMapEntry
+  vinnumalastofnun: StateMapEntry
+}
+const statesMap: StatesMap = {
+  employer: {
+    [ApplicationStates.EMPLOYER_WAITING_TO_ASSIGN_FOR_EDITS]:
+      ReviewSectionState.inProgress,
+    [ApplicationStates.EMPLOYER_APPROVE_EDITS]: ReviewSectionState.inProgress,
+    [ApplicationStates.VINNUMALASTOFNUN_APPROVE_EDITS]:
+      ReviewSectionState.complete,
+  },
+  vinnumalastofnun: {
+    [ApplicationStates.VINNUMALASTOFNUN_APPROVE_EDITS]:
+      ReviewSectionState.inProgress,
+  },
+}
+
+const EditInReviewSteps: FC<FieldBaseProps> = ({ application }) => {
+  const dob = getExpectedDateOfBirth(application)
+  const dobDate = dob ? new Date(dob) : null
+
+  const { formatMessage } = useLocale()
+
+  const steps = [
+    {
+      state: statesMap['employer'][application.state],
+      title: formatMessage(
+        parentalLeaveFormMessages.editFlow.employerApprovesTitle,
+      ),
+      description: formatMessage(
+        parentalLeaveFormMessages.editFlow.employerApprovesDesc,
+      ),
+    },
+    {
+      state: statesMap['vinnumalastofnun'][application.state],
+      title: formatMessage(parentalLeaveFormMessages.reviewScreen.deptTitle),
+      description: formatMessage(
+        parentalLeaveFormMessages.reviewScreen.deptDesc,
+      ),
+    },
+  ]
+
+  return (
+    <Box marginBottom={10}>
+      <Box
+        display={['block', 'block', 'block', 'flex']}
+        justifyContent="spaceBetween"
+      >
+        {dobDate && (
+          <Text variant="h4" color="blue400">
+            {formatMessage(
+              parentalLeaveFormMessages.reviewScreen.estimatedBirthDate,
+            )}
+            <br />
+            {format(dobDate, 'dd.MM.yyyy')}
```

Comment:
The format is stored in a constant [here](https://github.com/island-is/island.is/blob/c0c5e36c3f63d4d0a16421ff55f742541862056c/libs/shared/constants/src/lib/date.ts#L3).

Reply:
Nice, this should be fixed now.